plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/githubRepoPush.test.ts

```diff
@@ -495,4 +505,70 @@ describe('github:repo:push', () => {
 
     expect(enableBranchProtectionOnDefaultRepoBranch).not.toHaveBeenCalled();
   });
+
+  it('should call enableBranchProtectionOnDefaultRepoBranch with the correct values of dismissStaleReviews', async () => {
```

Comment:
Use [`it.each([false, true])`](https://jestjs.io/docs/api#testeachtablename-fn-timeout) instead? This makes it obvious that the test is repeated and identical in both cases besides this one boolean.

Reply:
I feel the first half is a bit redundant tbh, it's already tested multiple times above